L1Trigger/L1TTrackMatch/python/L1TrackSelectionProducer_cfi.py

```diff
@@ -25,7 +28,15 @@
   useDisplacedTracksDeltaZOverride = cms.double(-1.0), # override the deltaZ cut value for displaced tracks
   processSimulatedTracks = cms.bool(True), # return selected tracks after cutting on the floating point values
   processEmulatedTracks = cms.bool(True), # return selected tracks after cutting on the bitwise emulated values
-  debug = cms.int32(0) # Verbosity levels: 0, 1, 2, 3, 4
+  debug = cms.int32(0), # Verbosity levels: 0, 1, 2, 3, 4
+
+  useAssociationNetwork = cms.bool(True), #Enable Association Network
+  AssociationThreshold = cms.double(0.5), #Association Network threshold for PV tracks
+  AssociationGraph = cms.string("L1Trigger/L1TTrackMatch/data/Quantised_model_prune_iteration_9_associationModelgraph.pb"), #Location of Association Network model file
```

Comment:
I still find this does not necessary work well for me depending on if I am running in local or batch. I will try to look into it

Reply:
What looks to work for me is to include at the start of this cfi and the VertexProducer_cff:
`
import os 
CMSSW_BASE = os.getenv('CMSSW_BASE')
`
Then changing this line to be `  AssociationGraph = cms.string(CMSSW_BASE+"/src/L1Trigger/L1TTrackMatch/data/Quantised_model_prune_iteration_9_associationModelgraph.pb"), #Location of Association Network model file`

And in VertexProducer_cff:
`
        TrackWeightGraph = cms.string(CMSSW_BASE+"/src/L1Trigger/VertexFinder/data/Quantised_model_prune_iteration_9_weightModelgraph.pb"),
        PVZ0Graph = cms.string(CMSSW_BASE+"/src/L1Trigger/VertexFinder/data/Quantised_model_prune_iteration_9_patternModelgraph.pb"),
`

Reply:
Added these changes in